package.json

```diff
@@ -35,12 +35,12 @@
     "@angular/platform-browser": "12.1.1",
```

Comment:
What about these?

Reply:
We cant upgrade those until we upgrade the node version of our docker image